pkg/jobs/collector.go

```diff
@@ -302,7 +302,7 @@ func (jb *jobCollector) ApplyAndCollect(ctx context.Context, nodeName string) (s
 	}
 	nc, err := jb.loadNodeConfig(ctx, nodeName)
 	if err != nil {
-		return "", fmt.Errorf("loading node config: %w", err)
+		return "", fmt.Errorf("loading node config for %q: %w", nodeName, err)
```

Comment:
Nice small change! looks like a good opportunity to add a test case for it.

Reply:
@simar7 I agree with you, and tried to add it.
there were a lot of updates and I've created #432 

It seems like too many changes for such a small PR.
wdyt?

Reply:
Ok sounds good.